examples/interactivity/polygons/feature-click.html

```diff
@@ -93,24 +93,25 @@ <h2></h2>
         console.log('CLICK OUT', event.features);
       });
     });
-    
+
```

Comment:
Another :rainbow: , I don't know why there are so many spaces like this one, I see lots of PRs, including mine that removed them, but I don't know how they are introduced in the first place, it would be nice to get rid of them :slightly_smiling_face:

Reply:
I have checked my editor (Atom) to "Ensure Single Trailing Newline". @davidmanzanares and @IagoLast you should check also your config preferences.